src/shared/dropdown-list/dropdown-list.tsx

```diff
@@ -295,12 +297,16 @@ export const DropdownList = <T, V>({
                                     displaySize={"small"}
                                 />
                             )}
-                            <Label truncateType={itemTruncationType}>
-                                {itemTruncationType === "middle" &&
-                                hasExceededContainer(item)
-                                    ? renderTruncatedText(item)
-                                    : getOptionLabel(item)}
-                            </Label>
+                            {renderListItem ? (
+                                renderListItem(item)
```

Comment:
`ListItemSelector` has a fixed 3.5rem height. can you run this past UX to see if future dropdowns might need larger, dynamically sized items? can perhaps relax the fixed height if `renderListItem` is defined

Reply:
Will check with UX tmr